recipes/deco/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class DecoConan(ConanFile):
+    name = "deco"
+    license = "Apache 2.0 with LLVM exceptions"
```

Comment:
What's the official SPDX name for it?

Reply:
following "SPDX License Exceptions" in https://spdx.dev/ids/
and https://spdx.org/licenses/LLVM-exception.html
it should be `Apache-2.0 WITH LLVM-exception`

Reply:
So, please, use `LLVM-exception` as described as official short name. It makes easier to parse and find when using scripts. Also, white space is identified as license separator, so it's not allowed.

Reply:
already updated